src/nvc.c

```diff
@@ -110,6 +110,24 @@ static void missing_argument(const char *what, char **argv)
    fatal("%s option $bold$%s$$ requires an argument", what, argv[optind - 1]);
 }
 
+static void parse_cond_analysis_identifier(char *optarg)
+{
+   char *name = optarg;
```

Comment:
This does something a bit odd when the option has multiple `=`s. I think it would be better to just use `strchr` like this:

```c
char *eq = strchr(optarg, '=');
if (eq == NULL)
   fatal("$bold$--define$$ argument must be KEY=VALUE");

*eq = '\0';
conda_id_add(optarg, eq + 1);
```